packages/tupaia-web/src/features/Map/MapOverlaySelector/DesktopMapOverlaySelector.tsx

```diff
@@ -49,6 +49,7 @@ const Heading = styled(Typography).attrs({
 
 const Container = styled(MaxHeightContainer)`
   border-radius: 0 0 5px 5px;
```

Comment:
I wonder if we should save this border radius value somewhere (just the `5px`) since it gets used a lot. I know not strictly part of this PR, but just something I thought of

Reply:
hmmm we could do but I wonder if it would actually be helpful. I don't think that is something that would change in a theme for example.